mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -79,7 +79,7 @@ CLASS sg net/minecraft/entity/Entity
 	METHOD T invalidate ()V
 	METHOD U updateLogic ()V
 	METHOD X extinguish ()V
-	METHOD a inRenderRange (D)Z
+	METHOD a isInRenderRange (D)Z
 		ARG 0 range
```

Comment:
`distance` ? Not sure, but the argument you pass to the function isn't the range itself, but the distance that has to be checked against the range, right?